src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -34,7 +46,10 @@ const ResearchProfile = () => {
   const [projectRowsPerPage, setProjectRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
   const [registrationRowsPerPage, setRegistrationRowsPerPage] = useState(ROWS_PER_PAGE_OPTIONS[0]);
 
-  const currentCristinId = useSelector((store: RootState) => store.user?.cristinId) ?? '';
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const user = useSelector((store: RootState) => store.user)!; // If user has been empty this route would already be blocked
```

Comment:
Dette stemmer vel ikke her, da man kan gå på forskerprofilen til hvem som helst uten å logge inn 🤔 
```suggestion
  const user = useSelector((store: RootState) => store.user);
```

Reply:
Ser du har `user?` under her også, som betyr at du har lagt inn håndtering av at user er undefined, på tross av at det i linja over står at "ignorer den implisitte typen her, for jeg vet at denne alltid har verdi" 😅